valhalla/jawn/src/managers/SessionManager.ts

```diff
@@ -364,4 +365,36 @@ WHERE ${buildWhereClause("duration")}
       }))
     );
   }
+
+  async updateSessionFeedback(
+    sessionId: string,
+    rating: boolean
+  ): Promise<Result<null, string>> {
+    const { data, error } = await supabaseServer.client
```

Comment:
@kavinvalli Let's add auth check here. Also we need to select by org id too like :
eq("organization", this.orgId)

Reply:
Actually, just org id to select statement, we will cover auth check if there is no requests

Reply:
Done!